src/System.Runtime.Caching/ref/System.Runtime.Caching.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netfx-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netfx-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
+  <PropertyGroup>
+    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'netfx'">true</IsPartialFacadeAssembly>
```

Comment:
Why are we generating a partial facade for the reference assembly and not for the source? If we have the same identity we shouldn't do that. If we are planing on a facade we should have it in the ref and the source, since the way you have it is that the reference assembly will be a facade and the implementation will be a PlatformNotSupported assembly.

We should define wether we want to share the same identity or not (if not rename the assembly as @danmosemsft  suggests).

Reply:
Put another way @KKhurin if you keep the identity as is you should remove this line from the csproj here: we don't need a facade because you are the thing itself.

Reply:
fixed